packages/components/src/Notification/subcomponents/GenericNotification/_mixins.scss

```diff
@@ -5,9 +5,8 @@
 @import "~@kaizen/design-tokens/sass/border";
 @import "~@kaizen/design-tokens/sass/animation";
 
-$notification-vertical-padding: calc(
-  #{$spacing-6} - #{$border-solid-border-width}
-);
+$border-width: var(--border-width-1);
```

Comment:
```suggestion
--border-width: var(--border-width-1);
```

Reply:
As discussed, this one is a bit special since it's used in another scss variable, and it is set globally in the file. I've only been creating the css vars in scoped contexts to avoid name clashes.